bgcval2/custom_plots/PIControl_sst.py

```diff
@@ -174,7 +174,7 @@ def get_data(j, field='AMOC'):
 
     if not os.path.exists(fn):
         print('Does not exist:', fn)
-    shelve = shopen(fn)
+    shelve = shOpen(fn)
```

Comment:
context managre here too?

Reply:
Not worth the effort - don't use this code.